README.md

```diff
@@ -1,7 +1,7 @@
 [![CI Status](https://github.com/AY2021S1-CS2103T-F11-4/tp/workflows/Java%20CI/badge.svg)](https://github.com/AY2021S1-CS2103T-F11-4/tp/actions)
 [![codecov](https://codecov.io/gh/AY2021S1-CS2103T-F11-4/tp/branch/master/graph/badge.svg)](https://codecov.io/gh/AY2021S1-CS2103T-F11-4/tp)
 
-![Ui](docs/images/Ui.png)
+![Ui](docs/images/Ui_old.png)
```

Comment:
the file Ui_old seems to have been renamed to Ui in this PR

Reply:
missed this out. Should point to the correct one now